File: Miet/divD1/HW-1/G.cpp
/*
* @Author: kostya
* @Date:   2021-11-13 16:57:33
* @Last Modified by:   kostya
* @Last Modified time: 2021-11-13 17:02:25
*/

#include <string>
#include <iostream>

using namespace::std;

int main()
{
	int n, t;
	string s;
	cin >> n >> t >> s;
	while (t--)
	{
		for (int i=1; i < n; ++i)
			if (s[i] == 'G')
				swap(s[i], s[i - 1]);
	}
	cout << s << endl;
}

BGGBG
GBGGB

File: Miet/divD1/HW-1/G.py
# -*- coding: utf-8 -*-
# @Author: kostya
# @Date:   2021-11-13 17:02:53
# @Last Modified by:   kostya
# @Last Modified time: 2021-11-13 17:05:58

n, t = map(int, input().split())
s = input()
for i in range(t):
	s = s.replace('BG', 'GB')
print(s)
